backend/src/ApiServer/ApiServer.fs

```diff
@@ -176,6 +176,7 @@ let rollbarCtxToMetadata (ctx : HttpContext) : (Rollbar.Person * Metadata) =
   let clientVersion =
     try
       string ctx.Request.Headers["x-darklang-client-version"]
+      |> String.take 7
```

Comment:
I've tested this ensuring it's OK when such a header doesn't exist (i.e. was cautious of a null reference exception).